src/nodes/Heading.js

```diff
@@ -1,6 +1,22 @@
 import TipTapHeading from '@tiptap/extension-heading'
+import HeadingView from './HeadingView.vue'
+import { VueNodeViewRenderer } from '@tiptap/vue-2'
 
-const Heading = TipTapHeading.extend({
+// Same style as used by gitlab and github, both support unicode letters (incl. mark)
+const slugify = (str) => String(str).toLowerCase().replace(/[^\p{Letter}\p{Mark}\w\s-]/gu, '').trim().replace(/\s+/g, '-')
+
+const uniqueSlug = (slug, slugs) => {
```

Comment:
How it exactly works?
It's changing `slugs`, a side effect.
I guess it is not good :/